### ISSUE TITLE
The paga

### ISSUE THREAD

**Issue by yuanzhiyuan** [theislab/paga#25]
Is the connectitivity matrix computed by PAGA sum into 1 by raw/column. If not, Is it suitable to normalize the matrix to a valid transition matrix for Markov chain?

**Comment by falexwolf** [comment]
Yes, that's what happens in diffusion maps: https://scanpy.readthedocs.io/en/stable/generated/scanpy.tl.diffmap.html
